packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -45,7 +66,8 @@ const Collapse = React.forwardRef(function Collapse(props, ref) {
     children,
     classes,
     className,
-    collapsedHeight: collapsedHeightProp = '0px',
+    collapsedHeight = '0px',
```

Comment:
Since we're targetting v5 I would go ahead and remove this prop in favor of `collapsedSize`. This should simply default value branching significantly.